sample-config.yaml

```diff
@@ -7,6 +7,8 @@ rates_channel_id: <channel to post rates in>
 
 # OPTIONAL
 polling_delay: 60
-allowed_roles: <roles permitted to use bot commands>
-rates_urls: "http://arkdedicated.com/dynamicconfig.ini"
+allowed_roles: 
+  - <roles permitted to use bot commands>
+rates_urls: 
+  - "http://arkdedicated.com/dynamicconfig.ini"
```

Comment:
Just a thought, and perhaps this hasn't come up because of the comma-separated-list parsing that was implemented, but these should technically be array types, right? I know with only a single item they effectively get parsed into the exact same thing, but if you actually had multiple values for these keys, I think you'd ideally want them to be a YAML array for readability.

Reply:
I thought it best to permit a comma separated list, as that would be more intuitive to a non developer who is just hoping to spin up a discord bot without prior knowledge of YAML syntax

Reply:
Yeah I see where you're coming from with that, but YAML is pretty common, and it's likely best to just use a common /well-established file format than come up with some new one for the sake of onboarding new people. 

 That being said, it's all going to work either way thanks to [this](https://github.com/FM-17/poglink/blob/08ea6d2c0233976efc7cac07f2c777fe269eadfb/poglink/bot.py#L35-L43) (and the other location in `setup_argparse` where this snippet is implemented). I just personally find the YAML array syntax easier on the eyes when there's more than one lengthy item to be provided. And although you support "proper" lists as well as comma separated syntax, if you had to pick one as a sample, I'd just favor the "proper" one, and leave the comma separated option as a happy surprise for someone if they thought to do it that way.

Reply:
unintentionally reacted with a thumbs down here. hope you took it to heart. 

I do agree though, and would be open to using YAML array syntax by default